### PR TITLE
Automated PyPI Wheels for Windows on a new release

### PR DESCRIPTION
**Pull request by geographika**
This pull request automatically uploads Python MapScript Wheels for Windows to https://pypi.org/project/mapscript/ on each new MapServer release. 

A release is defined by a new tag on the master branch e.g. rel-7-4-3, and will be created for each [MapServer Release](https://github.com/mapserver/mapserver/releases)

Currently Appveyor builds for the following Python versions, so each of these will have a Wheel created:

- Python 2.7 x32
- Python 2.7 x64
- Python 3.6 x64

It may be time to also add in a Python 3.7 environment. 

I've tested the process on my branch, with tags on master, and uploading to https://test.pypi.org/manage/project/mapscript/release/7.6.0/

In addition a "source distribution" is created that can be used to automatically generate MapScript API documentation (updates still ongoing).

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -10,6 +10,12 @@ cache:
   - '%APPVEYOR_BUILD_FOLDER%\swigwin-4.0.1.zip'
   
 environment:
+
+  global:
+    TWINE_USERNAME: geographika
+    TWINE_PASSWORD:
+      secure: 6yuYlaTUIL8SRBakWTke8w==
```
Comment (rouault) [inline appveyor.yml]: not sure how appveyor handles encrypted secret, but I would suspect that the secret should be encrypted with something specific to https://ci.appveyor.com/project/MapServer/mapserver and not your own appveyor account

Comment (geographika) [reply appveyor.yml]: This is an Appveyor encrypted secret of my PyPI password (Twine is the Python library that uploads to PyPI).  I have permissions to upload new packages to https://pypi.org/project/mapscript/. However, I can't add new maintainers (e.g. a new "mapserver" user) though as I'm not an admin. Maybe @tomkralidis does?

**Comment by geographika** [comment]
> 
> 
> @geographika looks like I don't have the ability to manage rights either. @kiorky: are you able to either 1./ provide manage rights to us per this ticket 2./ Add the `mapserver` user to the PyPI mapscript maintainers? Option 1 preferred for longer term sustainability (and less noise to yourself).

We will need to create a new PyPI user *mapserver* at https://pypi.org/account/register/
@sdlime - is there a communal mapserver email account we could use to register this user?

**Comment by kiorky** [comment]
No problem, but please create that `mapserver` user :).

**Comment by geographika** [comment]
Thanks @kiorky - mapserver user created at https://pypi.org/user/mapserver/

**Comment by kiorky** [comment]
done

**Comment by geographika** [comment]
> 
> 
> done

Thanks @kiorky !

**Comment by geographika** [comment]
The Python package will now be created using the new mapserver user. 
@jmckenna  - unless there are any objections I'd like to merge this ready for the 7.6 beta releases.

**Comment by jmckenna** [comment]
no objections here.  marking as 7.6.0...

**Comment by jmckenna** [comment]
@geographika thanks for this!

**Comment by geographika** [comment]
@kiorky - is it possible to make the mapserver user an owner of the mapscript package?
A development version of mapscript accidentally got deployed to PyPI and it would be good to delete it - https://pypi.org/project/mapscript/7.7.0/

**Comment by kiorky** [comment]
dist is deleted, remember that you cant reupload a 7.7.0 , next version will be 8.8.0

**Comment by geographika** [comment]
@kiorky - thanks! Yes the next release should be 7.6.1 or 8.0.

**Pull request merged**